search/query.go

```diff
@@ -227,7 +229,25 @@ func createResultsQuery(resultTypes []string, q Query, docIds []string) elastic.
 }
 
 func NewResultsSearchRequest(options SearchRequestOptions) *elastic.SearchRequest {
-	fetchSourceContext := elastic.NewFetchSourceContext(true).Include("mdb_uid", "result_type", "title")
+	fetchSourceContext := elastic.NewFetchSourceContext(true).Include("mdb_uid", "result_type")
+
+	titleAdded := false
+	contentAndDateAdded := false
+	//	This is a generic imp. that supports searching tweets together with other results.
+	//	Currently we are not searching for tweets together with other results but in parallel.
+	for _, rt := range options.resultTypes {
+		if !contentAndDateAdded && rt == consts.ES_RESULT_TYPE_TWEETS {
+			fetchSourceContext.Include("content").Include("effective_date")
+			contentAndDateAdded = true
+		} else if !titleAdded {
+			fetchSourceContext.Include("title")
```

Comment:
Why do we need title in source? Actually I don't remember why.

Reply:
we use title for all other results beside tweets